day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,24 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
+    @toppings.push("lettuce")
+  end
+
+  def remove_topping
+    @toppings.delete("cheese")
+  end
+
+  def change_protein
+    protein.replace("steak")
```

Comment:
Why are you able to use protein without the @symbol here?

Reply:
I want to say that it's because that value of protein is just being read and not actually changed. I ran some tests with pry trying to read the values, but I keep getting an error message about burrito not being a method rather than it giving me the values.
![Screen Shot 2020-10-03 at 5 02 21 PM](https://user-images.githubusercontent.com/52386927/95003206-352a5000-059a-11eb-9a72-f932646759d3.png)

Reply:
I did find that I was able to remove the @ symbol from all the methods and receive the same result as well, which I found interesting. I'm continuing to look into it, but at least wanted to respond to you before class on monday.